inject-java/src/test/groovy/io/micronaut/aop/compile/AroundCompileSpec.groovy

```diff
@@ -988,6 +988,92 @@ class TestInterceptor implements Interceptor {
         context.close()
     }
 
+    void 'test interceptor on an event'() {
+        given:
+        ApplicationContext context = buildContext('''
+package test;
+
+import java.lang.annotation.*;
+import io.micronaut.aop.*;
+import io.micronaut.context.annotation.Type;
+import io.micronaut.context.event.ApplicationEventListener;
+import io.micronaut.context.event.ApplicationEventPublisher;
+import io.micronaut.core.annotation.Indexed;
+import io.micronaut.core.annotation.Internal;
+import jakarta.inject.*;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+import io.micronaut.aop.simple.*;
+import jakarta.inject.Singleton;
+
+class TheEvent {
+}
+
+@Singleton
+class MyBean {
+
+    private final ApplicationEventPublisher applicationEventPublisher;
+    long count = 0;
```

Comment:
should you verify in the test that this counter is increased? You are verifying only the interceptor counter right?